app/iht/views/iht_main_template.scala.html

```diff
@@ -50,7 +49,8 @@
 headingClass: String = "",
 headingName: String = "",
 isRegistration: Boolean = false,
-isUserResearchBannerVisible:Boolean = false
+isUserResearchBannerVisible:Boolean = false,
+highlightBox:Boolean = false
```

Comment:
Call it something like isHighlightBoxVisible

Reply:
Done